### ISSUE TITLE
Deprecated-warnings for Int + String in exercise `expressions`

### ISSUE THREAD

**Issue by snctfd** [lunduniversity/introprog#473]
The statements in Uppgift 9 in the `expressions` exercise trigger the following warning in Scala v2.13.3:

```
warning: method + in class Int is deprecated (since 2.13.0): Adding a number and a String is deprecated. Use the string interpolation `s"$num$str"`
```

This confused some of my students that were unsure if they had to care about warnings. Seeing as lecture 1 is perhaps a bit early to introduce string interpolation, I propose that a notice is added to this specific Uppgift to let students know that they don't have to care about any warnings they get.

**Comment by jos-rosenqvist-3812** [comment]
This also affects the given solution for task 8 in `programs` and the given code in task 9 in `programs`.

**Comment by bjornregnell** [comment]
@snctfd Confusion and wonder is a good starting-point for learning :) Thanks for spotting this. Should also explain somewhere what depcreation means. It is good to understand that a programming langugae and its standard library is a continuous flow of improvements...

**Comment by IowU** [comment]
If you want to change your code so to match the new syntax required by scala >= 2.13 you can change the for loop in Uppgift9

```
for(i<-1 to 10) print(i + ", ")
```

to

```
for(i<-1 to 10) print(s"$i, ")
```
since now it requires the s interpolation string. You can apply a similar change to the other cases

**Comment by bjornregnell** [comment]
Yes, a string interpolation should be used here. Feel free to come with a PR if you like :) Otherwise I'll fix this when we later this spring convert to Scala 3.

**Comment by IowU** [comment]
I'm still learning git, so I'd prefer not to make any PR yet. I hope my suggestion will be of help

**Comment by bjornregnell** [comment]
Ok that's fine. Thanks for your help and feedback!
